tests/test_edgeql_data_migration.py

```diff
@@ -2860,6 +2867,43 @@ async def test_edgeql_migration_eq_14(self):
             }],
         )
 
+    async def test_edgeql_migration_eq_14b(self):
+        # Same as above, except POPULATE and inspect the query
+        await self.migrate(r"""
+            type Base;
+
+            type Derived extending Base {
+                property foo -> str;
+            }
+        """)
+
+        await self.start_migration(r"""
+            type Base {
+                # move the property earlier in the inheritance
+                property foo -> str;
+            }
+
+            type Derived extending Base {
+                overloaded required property foo -> str;
+            }
+        """, populate=True)
+
+        await self.assert_describe_migration({
+            'confirmed': [
+                textwrap.dedent("""\
```

Comment:
I wonder if we should maybe teach `assert_describe_migration` to dedent stuff automatically.

Reply:
Yeah, that's a good idea. I went and did it.